source/configuration_manual/authentication/oauth2.rst

```diff
@@ -211,6 +211,14 @@ The following algorithms are supported
 There is currently no support for EdDSA algorithms.
 ES supports any curve supported by OpenSSL for this purpose.
 
+OpenID.Discovery
+****************
+
+.. versionadded:: v2.3.16
+
+Support for `RFC 7628 <https://datatracker.ietf.org/doc/html/rfc7628#section-3.2.2>`__  OpenID Discovery (OIDC) can be achieved with
+``openid_configuration_url`` setting. Setting this causes Dovecot to report OIDC configuration URL as ``openid-configuration`` element in error JSON.
+
```

Comment:
Should we also reference the example in section 4.3? I think it might potentially help client developers.

Reply:
...how?

Reply:
also one would hope client developers would read the whole rfc.

Reply:
If it is not handy, lets indeed hope everybody reads the RFC closely. Im fine with that also.